config/config.go

```diff
@@ -46,50 +46,46 @@ const (
-	osPatchEnabledDefault     = false
-	debugEnabledDefault       = false
-
-	configDirWindows        = `C:\Program Files\Google\OSConfig`
-	configDirLinux          = "/etc/osconfig"
-	osPatchStateFileWindows = configDirWindows + `\osconfig_patch.state`
-	osPatchStateFileLinux   = configDirLinux + "/osconfig_patch.state"
-	taskStateFileWindows    = configDirWindows + `\osconfig_task.state`
-	taskStateFileLinux      = configDirLinux + "/osconfig_task.state"
-	restartFileWindows      = configDirWindows + `\osconfig_agent_restart_required`
-	restartFileLinux        = configDirLinux + "/osconfig_agent_restart_required"
+	osInventoryEnabledDefault      = false
+	guestPoliciesEnabledDefault    = false
+	taskNotificationEnabledDefault = false
+	debugEnabledDefault            = false
+
+	configDirWindows     = `C:\Program Files\Google\OSConfig`
+	configDirLinux       = "/etc/osconfig"
+	taskStateFileWindows = configDirWindows + `\osconfig_task.state`
+	taskStateFileLinux   = configDirLinux + "/osconfig_task.state"
+	restartFileWindows   = configDirWindows + `\osconfig_agent_restart_required`
+	restartFileLinux     = configDirLinux + "/osconfig_agent_restart_required"
 
 	osConfigPollIntervalDefault = 10
 )
 
 var (
-	resourceOverride = flag.String("resource_override", "", "The URI of the instance this agent is running on in the form of `projects/*/zones/*/instances/*`. If omitted, the name will be determined by querying the metadata service.")
-	endpoint         = flag.String("endpoint", prodEndpoint, "osconfig endpoint override")
-	oauth            = flag.String("oauth", "", "path to oauth json file")
-	debug            = flag.Bool("debug", false, "set debug log verbosity")
-	stdout           = flag.Bool("stdout", false, "log to stdout")
+	endpoint = flag.String("endpoint", prodEndpoint, "osconfig endpoint override")
+	debug    = flag.Bool("debug", false, "set debug log verbosity")
+	stdout   = flag.Bool("stdout", false, "log to stdout")
 
 	agentConfig   = &config{}
 	agentConfigMx sync.RWMutex
 	version       string
 )
 
 type config struct {
-	osInventoryEnabled, osPackageEnabled, osPatchEnabled, debugEnabled                    bool
+	osInventoryEnabled, guestPoliciesEnabled, taskNotificationEnabled, debugEnabled       bool
 	svcEndpoint, googetRepoFilePath, zypperRepoFilePath, yumRepoFilePath, aptRepoFilePath string
 	numericProjectID, osConfigPollInterval                                                int
 	projectID, instanceZone, instanceName, instanceID                                     string
 }
 
 func (c *config) parsePreRelease(features string) {
 	for _, f := range strings.Split(features, ",") {
-		f = strings.TrimSpace(f)
+		f = strings.ToLower(strings.TrimSpace(f))
 		switch f {
-		case "ospatch":
-			c.osPatchEnabled = true
-		case "ospackage":
-			c.osPackageEnabled = true
+		case "tasknotification", "ospatch": // ospatch is the legacy flag
+			c.taskNotificationEnabled = true
+		case "ospackage", "guestpolicies":
```

Comment:
nit: add a similar comment for ospackage to identify it as a legacy flag similar to the comment above for ospatch?